metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -387,6 +410,47 @@ def populate_view_lineage_with_sql_parsing(
                 )
             )
 
+    def populate_snapshot_lineage_with_sql_parsing(
```

Comment:
this method looks really similar to `populate_view_lineage_with_sql_parsing`

is the main reason that it's separate because the reporting stuff is different? or are there other differences that I'm missing?